README.md

```diff
@@ -45,7 +45,7 @@ git clone --branch openy-gc-builds \
   git@github.com:fivejars/docksal-for-openy.git .docksal
 git clone git@github.com:fivejars/openy_gated_content.git
```

Comment:
This line is not needed if the official repo is used

Reply:
instead of using local folder ( which fails composer if not available ) there is a way to use git repository without any services as packagist or openy mirror, where there are cache layers, killing CI